config/routes.rb

```diff
@@ -82,6 +82,7 @@ def set_up_flipper
       end
     end
 
+    resources :partnerbases, only: %i(edit update)
```

Comment:
@seanmarcia what are you thoughts on changing the routing to be like:
```
UPDATE http://localhost:3000/org/partners/262/profile
GET http://localhost:3000/org/partners/262/profile/edit
```
I think this plays nicely with the concept that the information shown is for a profile.

Reply:
made it a nested route

Reply:
also changed it to profile rather than partnerbase